src/workerd/server/tests/python/env-param/env.wd-test

```diff
@@ -14,7 +14,7 @@ const unitTests :Workerd.Config = (
           ),
         ],
         compatibilityDate = "2024-01-15",
-        compatibilityFlags = ["python_workers"],
```

Comment:
Would it make more sense to duplicate all of those tests to run against the old flag as well?

Reply:
Yeah it would. I think @garrettgu10 has offered to do this in a followup.